ci-operator/step-registry/ipi/conf/aws/user-min-permissions/ipi-conf-aws-user-min-permissions-ref.yaml

```diff
@@ -29,6 +29,10 @@ ref:
     - name: FIPS_ENABLED
       default: "false"
       documentation: "Indicates a FIPS-enabled cluster, to disable FIPS host checks in the installer."
+    - name: OPENSHIFT_INSTALL_AWS_PUBLIC_ONLY
+      default: ""
+      documentation: |-
+        Whether to use only public subnets for AWS. Implies no NAT Gateways.
```

Comment:
Suggest to add a note that indicates `OPENSHIFT_INSTALL_AWS_PUBLIC_ONLY` will be used by `create permissions-policy` validation.

Reply:
Done